src/pyodide/internal/python.js

```diff
@@ -350,7 +362,7 @@ export function mountLib(pyodide, requirements, isWorkerd) {
   sys.destroy();
 }
 
-export async function loadPyodide(context, lockfile, indexURL) {
```

Comment:
This will be used for snapshot upload, I know it's not used but I'd rather you left it in here so I can avoid dealing with merge conflicts later.

Reply:
It won't be available when calling `loadPyodide` at top level, so I think the snapshot upload will need to go somewhere else.